src/compiler/binder.ts

```diff
@@ -3467,7 +3466,7 @@ function createBinder(): (file: SourceFile, options: CompilerOptions) => void {
         let includes = SymbolFlags.None;
         let excludes = SymbolFlags.None;
         // Method-like
-        if (isFunctionLikeDeclaration(getAssignedExpandoInitializer(declaration)!)) {

```

Comment:
I added `| undefined` to `isFunctionLikeDeclaration` as that's a possible runtime value so now the non-null assertion isn't needed